cpp/dolfinx/mesh/Geometry.h

```diff
@@ -328,22 +328,15 @@ create_subgeometry(const Topology& topology, const Geometry<T>& geometry,
   // Get the sub-geometry dofs owned by this process
   auto x_index_map = geometry.index_map();
   assert(x_index_map);
-  std::vector<std::int32_t> subx_to_x_dofmap
-      = common::compute_owned_indices(sub_x_dofs, *x_index_map);
+
   std::shared_ptr<common::IndexMap> sub_x_dof_index_map;
+  std::vector<std::int32_t> subx_to_x_dofmap;
   {
     std::pair<common::IndexMap, std::vector<int32_t>> map_data
```

Comment:
Could `std::tie` make this code easier to read and avoid the `std::move` at the end of the scope?

Reply:
I don't think so, unless I'm missing something